apps/src/aiComponentLibrary/suggestedPrompt/suggested-prompt.module.scss

```
@@ -1,38 +1,56 @@
-@import "color.scss";
-@import "font.scss";
-@import "@code-dot-org/component-library-styles/typography.module";
+@use '@code-dot-org/component-library-styles/mixins';
+@import '@cdo/apps/lab2/views/components/layout/variables.scss';
```

Sorry another nit! `@use` is the preferred way to import stuff in Sass now ([ref](https://sass-lang.com/documentation/at-rules/use/#differences-from-import)). You'll also need to update line 13 with `variables.$default-border`.
```suggestion
@use '@cdo/apps/lab2/views/components/layout/variables.scss' as variables;
```

---

updated!